sdk/eidolon_ai_sdk/agent/simple_agent.py

```diff
@@ -98,21 +114,25 @@ class SimpleAgentSpec(BaseModel):
     agent_refs: List[str] = []
     actions: List[ActionDefinition] = [ActionDefinition()]
     cpu: AnnotatedReference[APU] = None
```

Comment:
Since you added the before validator, you will can remove cpu from the base model definition

Reply:
Oh -- I didn't know that...